internal/commands/vaultsecrets/gatewaypools/displayer.go

```diff
@@ -8,16 +8,27 @@ import (
 	"github.com/hashicorp/hcp/internal/pkg/format"
 )
 
+type gatewayPoolWithIntegrations struct {
+	GatewayPool  *preview_models.Secrets20231128GatewayPool
+	Integrations []string
+}
+
 type displayer struct {
-	gatewayPools []*preview_models.Secrets20231128GatewayPool
+	gatewayPools []*gatewayPoolWithIntegrations
+
+	// showIntegrations is used to determine if the integrations should be shown
+	// This is used only for the read command where the integrations associated with
+	// the gateway pool is also displayed
+	showIntegrations bool
```

Comment:
You could replace this bool with `Integrations *[]string` if you wanted.  It's ok as it is though.